cmake/std/atdm/cts1/custom_builds.sh

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
Just an FYI for later work to update the 'tlcc2' env for SPARC ...

Interestingly, it looks like SPARC has not installed an Intel 18 env for the TLCC2 machines.  For example, 'chama' only has the modules:

```
sparc-dev/intel-17.0.1_openmpi-1.10.5
sparc-dev/intel-19.0.5_openmpi-4.0.1
```

and does **not** have a `spack-dev/intel-18.0.2_openmpi-2.0.3`env!

Therefore, it looks like the atdm/tlcc2/ will have its own `custom_builds.sh` script which is different than this one for atdm/cts1/build_scirpts.sh.